docs/3_development/1_development-flow/3_review.md

```diff
@@ -11,13 +11,13 @@ Document everything, communicate with your team transparently, review together.
 
 - Ensure readability is retained in code e.g. complex nested ternary operators should have a comment note
 - Ensure adequate testing exists where needed
```

Comment:
I have a need to define `adequate` 🙈 but realized that is out of scope - maybe just link to QA docs?

Reply:
https://github.com/ChainSafe/engineering-handbook/pull/107/commits/4781b6d47a7e258410bf1a82d7dabe9bdf9e09e9